HunterPie.Core/Game/IGame.cs

```diff
@@ -19,9 +19,12 @@ public interface IGame : IDisposable
 
     public int Deaths { get; }
 
+    public int QuestLife { get; }
```

Comment:
I assume you named this property as `QuestLife` because that's it's internal name in MHR, but keep in mind HunterPie is agnostic to the game it's hooking to, so keeping a name that's common for **every** game is the ideal here. Also `QuestLife` is a pretty bad name as it is really not descriptive to what it actually is.
```suggestion
    public int MaxDeaths { get; }
```